docs/BUILD.md

```diff
@@ -16,36 +17,62 @@ Flix can be built and run from within the IntelliJ IDEA IDE which is useful for
 
 Traverse to `main/src/ca/uwaterloo/flix/Main.scala`
 
-Open `Main.scala`, right-click on `def main` and select `Run 'Main'`. This runs the Flix REPL.
+Open `Main.scala`, right-click on `def main` and select `Run 'Main'`. This runs
+the Flix REPL.
+
+To compile and run a Flix file with a `main` function, right-click on the main
+function in `Main.scala` and choose "Modify Run Configuration".
 
-To compile and run a Flix file with a `main` function, right-click on the main function in `Main.scala` and choose "Modify Run Configuration".
 Here enter the file name in the "Program arguments" field.
 
 ### Running tests
 
 Open `TestAll.scala`, right click on `class TestAll` and select `Run 'TestAll'`
 
+### Testing VSCode
+
+Add a `gradle.properties` file to the project root with:
+
+```
+dev.flix.vscode.project=C:/tmp/flix
```

Comment:
Can this use the temp directory from the path to be more OS-neutral?

Reply:
Does it matter? At least its clear what it is supposed to be.